build/scripts/Generate-ThirdPartyNotices.ps1

```diff
@@ -0,0 +1,14 @@
+[CmdletBinding()]

```

Comment:
This file only gets called on the release build though, and that script isn't a part of this repo...?

Reply:
Indeed. Eventually, it will be. We have an outdated version that's rotted in the repo, which i could add this to as a token gesture, but ...

Reply:
meh idc.